packages/redis/redis.conf

```diff
@@ -0,0 +1,5 @@
+# Directory to store dump.rdb
+dir /app/redis
+
+# Save the data to disk every 60 seconds if at least 1 key changed
+save 60 1
```

Comment:
How did you decide that this is a good interval? Is this a common default value? Why not something lower?

Reply:
As discusses we leave it at 60 seconds